### ISSUE TITLE
Internal Users should still be able to see Approved Prime Admin Requests

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/cas-registration#956]
#### Describe the task

We recently introduced an undesired behaviour where internal users ("cas_admin") can no longer view a prime admin request (UserOperator record) in the "Operators" table (that's actually UserOperators) once the request has been Approved. IRC staff have indicated that they still want to be able to view these requests once approved.

The desired behaviour for MVP is this:
- The view shows all pending requests if the operator does not have an approved admin
- Once an admin is approved by CAS, only that request record is shown in the view for that operator
- To achieve this we'll likely need a union of the what the current implementation is fetching (all pending requests for operators without an admin) and something like this: all user_operator records where the status is 'Approved' & the verified_by foreign key points to a user with the app_role of cas_admin or cas_analyst <-- this should populate the table with all records where a CAS user approved them

#### Acceptance Criteria

- [x] all admin requests that were "Approved" by CAS should appear in the UserOperator table (in the "operators" dashboard tile)
- [x] subsequent requests that are approved by the operator admin do not show up in the CAS view
- [x] request records for operators without an approved admin still show up in the table
- [ ] stretch goal (timeboxed): is there an easy way to add filtering to the status column so that internal users have the choice to hide/show Approved requests? **<- I think this should be in a follow up card**

**Comment by dleard** [comment]
Adding some thoughts I posted in a teams thread:

Some thoughts:
- The view could get very cluttered with approvals without the ability to sort or filter the results. 
- Are we only showing the approved admins that CAS approved? 
- If so we'll have to filter the request to only show results where the verified_by foreign key points to a user with the app_role of cas_admin or cas_analyst
- How do we handle stale data in this view? (ie: the user that CAS approved is no longer an admin because they left the company or their role changed).
- Does CAS want to see all approved users for an operator? Or just the one they approved as above?

I'll add a card to explore these questions further, for now I'll update the description of this card with some context / suggestions on implementation
